calculator-project/src/App.js

```diff
@@ -0,0 +1,57 @@
+import "./App.css";
+import { Component } from "react";
+import ResultInputs from "./Components/ResultInputs";
+import KeyPadInput from "./Components/KeyPadInput";
+
+class App extends Component {
+  state = { result: "" };
```

Comment:
I think it's better to have your operator in your state than it is to just have result and using eval